irida_import/tests/unit/test_irida_import.py

```diff
@@ -129,7 +129,7 @@ def test_get_samples(self, imp, setup_json):
         """
```

Comment:
I think we should have tests for the different scenarios for content type.
I could see 3 new tests that are similar to the existing test here `test_get_samples`. 
One test for each of fasta, fastq, and fast5, making sure the content type is set correctly on the sample file  objects for each of them.

Reply:
Sure I will get those added. Thanks!

Reply:
Added in those tests plus a 4th one which throws an exception if the content type can't be detected from the href due to the href being invalid in 91e1cc8a38fb6f08370cab677e67d670ad3db348.

Reply:
These tests look great!